pallets/ajuna-awesome-avatars/src/mock.rs

```diff
@@ -120,8 +154,14 @@ impl pallet_ajuna_awesome_avatars::Config for Test {
 	type FeeChainMaxLength = AffiliateMaxLevel;
 	type AffiliateHandler = Affiliates;
 	type TournamentHandler = Tournament;
-	type FeeHandler =
-		NativeGameFeeHandler<MockAccountId, Balances, WithdrawNative<Test>, Affiliates, Tournament>;
+	type FeeHandler = NativeGameFeeHandler<
+		MockAccountId,
+		Balances,
+		WithdrawCreditOrVoucher<WithdrawNative<Test, ()>, MockVoucherHandler>,
```

Comment:
Instead of using the mock here, can't we still just use the `WithdrawNative` here instead of adding the voucher?